src/core_plugins/timelion/public/panels/timechart/schema.js

```diff
@@ -242,11 +242,7 @@ module.exports = function timechartFn(Private, config, $rootScope, timefilter, $
             });
           }
 
-          try {
-            $scope.plot = $.plot(canvasElem, _.compact(series), options);
-          } catch (e) {
-            setTimeout(drawPlot, 500);
-          }
+          $scope.plot = $.plot(canvasElem, _.compact(series), options);
```

Comment:
what error was cached here ? just plot throwing because width or height was set to 0 ?